Intersect.Server/Entities/Combat/DoT.cs

```diff
@@ -99,12 +98,12 @@ public void Tick()
             var aliveAnimations = new List<KeyValuePair<Guid, sbyte>>();
```

Comment:
Why are we even using sbyte if it's just only directions that are used as the value?

Reply:
```csharp
var aliveAnimations = new List<KeyValuePair<Guid, Direction>>();